src/EA.Weee.Web/Areas/Admin/Views/Aatf/ManageAatfs.cshtml

```diff
@@ -16,15 +14,60 @@
         })
     </script>
 }
+
+<style>
```

Comment:
Can we add this to the weee-page.scss :)

Reply:
Moved